packages/calcite-components/src/components/filter/filter.scss

```diff
@@ -1,3 +1,17 @@
+/**
```

Comment:
@alisonailea I don't get why this PR is adding these vars. I know you said the doc outlined it but...

1. The PR commit/merge message doesn't mention adding css vars (a feature) to the filter.
2. This is duplicate work already being done as part of #8722

IMO from a conventional commit perspective these changes to other components seem out of scope.

If they were only added in #8722 which has the message `feat(filter): add component tokens #7180` it would make more sense.

cc @jcfranco

Reply:
These are all going into an Epic that I was under the impression would be squash merged requiring a cleanup of the final commit message anyway. Am I wrong?

Reply:
Yeah thats a good point. I guess it just seems a bit out of order and redundant.